core/vm/actor/builtin/types/miner/monies.hpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/builtin/states/miner/miner_actor_state.hpp"
+#include "common/outcome.hpp"
+#include "common/smoothing/alpha_beta_filter.hpp"
+#include "const.hpp"
+#include "primitives/types.hpp"
+#include "vm/actor/builtin/types/miner/policy.hpp"
+#include "vm/runtime/runtime.hpp"
+#include "vm/version/version.hpp"
```

Comment:
Please use auto-formatting to make an alphabetical order

Reply:
fixed